deploy/000_deploy_postage.ts

```diff
@@ -1,29 +1,48 @@
-import { HardhatRuntimeEnvironment } from 'hardhat/types';
 import { DeployFunction } from 'hardhat-deploy/types';
+import { networkConfig, developmentChains, deployedBzzData } from '../helper-hardhat-config';
 
-const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
-  const { deployments, getNamedAccounts } = hre;
-  const { deploy, execute, read } = deployments;
-
+const func: DeployFunction = async function ({ deployments, getNamedAccounts, network, ethers }) {
+  const { deploy, execute, read, log } = deployments;
   const { deployer, oracle, redistributor } = await getNamedAccounts();
 
-  const token = await deploy('TestToken', {
-    from: deployer,
-    args: [],
-    log: true,
-  });
+  let token = null;
+  if (developmentChains.includes(network.name)) {
+    token = await deploy('TestToken', {
+      from: deployer,
+      args: [],
+      log: true,
+    });
+  }
+
+  if (network.name == 'testnet') {
```

Comment:
This can be simplified to:
```
if (network.name == 'mainnet' || network.name == 'testnet') {
    token = await ethers.getContractAt(deployedBzzData[network.name].abi, deployedBzzData[network.name].address);
}
```
  
Then the if below can be removed.

Reply:
lol, obviously. did some refactoring and didnt even realize I ended with same 2 lines. Fixing this, thnx